SoftLayer/managers/network.py

```diff
@@ -388,6 +388,17 @@ def get_registrations(self):
 
         return self.account.getSubnetRegistrations(mask=mask)
 
+    def get_registration_details(self):
```

Comment:
Move this function to the registration manager.
Also allow this function to take in a `mask` parameter, and if it is None, set it to the default you have below.